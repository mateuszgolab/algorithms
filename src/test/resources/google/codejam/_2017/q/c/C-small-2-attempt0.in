100
4 2
5 2
6 2
1000 1000
1000 1
999999 477206
634699 496091
999999 524288
3 2
500000 1
524227 454645
500000 259081
325062 299901
1000000 499998
500000 237856
694787 604206
4 1
1000000 262143
500000 262144
1000000 524288
433410 402113
942608 754867
1000000 262144
663576 628750
999999 2
1000000 2
431155 429377
419406 329215
464494 404132
866735 863323
629114 509029
999999 475711
500000 128
963436 833369
2 2
255081 202696
458137 449444
999999 499997
999999 128
1000000 479714
1000000 379856
582807 496814
722124 695656
861605 777704
1000000 999999
339352 332533
999999 499999
1000000 127
999999 127
935184 918351
500000 500000
1000000 499999
500000 249999
957467 857082
500000 131072
999999 524287
1000000 475712
500000 199892
500000 131071
548568 527590
500000 249998
816856 770909
568110 560694
507707 388810
375175 334817
659090 539234
500000 127
500000 237857
999999 262143
1000000 475713
899322 793030
5 1
884624 748492
2 1
279333 246775
815280 687852
999999 1
1000000 524287
999999 262144
1000000 1
538341 528121
500000 2
1000000 500000
1000000 1000000
1 1
500000 499999
531220 468472
668063 658333
500000 262143
500000 250000
999999 999998
999999 475712
1000000 128
394765 370150
395574 374772
999999 499998
999999 999999
3 1
427865 331594
999999 377257
