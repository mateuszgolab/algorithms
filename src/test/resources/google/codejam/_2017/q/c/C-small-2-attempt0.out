Case #1: 1 0
Case #2: 1 0
Case #3: 1 1
Case #4: 0 0
Case #5: 500 499
Case #6: 1 0
Case #7: 0 0
Case #8: 0 0
Case #9: 0 0
Case #10: 250000 249999
Case #11: 0 0
Case #12: 1 0
Case #13: 0 0
Case #14: 1 0
Case #15: 1 1
Case #16: 0 0
Case #17: 2 1
Case #18: 3 2
Case #19: 0 0
Case #20: 0 0
Case #21: 0 0
Case #22: 0 0
Case #23: 1 1
Case #24: 0 0
Case #25: 249999 249999
Case #26: 250000 249999
Case #27: 0 0
Case #28: 0 0
Case #29: 0 0
Case #30: 0 0
Case #31: 0 0
Case #32: 1 1
Case #33: 1953 1952
Case #34: 0 0
Case #35: 0 0
Case #36: 0 0
Case #37: 0 0
Case #38: 1 0
Case #39: 3906 3905
Case #40: 1 0
Case #41: 1 1
Case #42: 0 0
Case #43: 0 0
Case #44: 0 0
Case #45: 0 0
Case #46: 0 0
Case #47: 1 0
Case #48: 7812 7811
Case #49: 7812 7811
Case #50: 0 0
Case #51: 0 0
Case #52: 1 0
Case #53: 1 0
Case #54: 0 0
Case #55: 1 1
Case #56: 1 0
Case #57: 1 1
Case #58: 1 1
Case #59: 3 2
Case #60: 0 0
Case #61: 1 0
Case #62: 0 0
Case #63: 0 0
Case #64: 0 0
Case #65: 0 0
Case #66: 0 0
Case #67: 3905 3905
Case #68: 1 0
Case #69: 3 2
Case #70: 1 0
Case #71: 0 0
Case #72: 2 2
Case #73: 0 0
Case #74: 1 0
Case #75: 0 0
Case #76: 0 0
Case #77: 499999 499999
Case #78: 1 0
Case #79: 1 1
Case #80: 500000 499999
Case #81: 0 0
Case #82: 125000 124999
Case #83: 1 0
Case #84: 0 0
Case #85: 0 0
Case #86: 0 0
Case #87: 0 0
Case #88: 0 0
Case #89: 1 0
Case #90: 1 0
Case #91: 0 0
Case #92: 1 0
Case #93: 3906 3905
Case #94: 0 0
Case #95: 0 0
Case #96: 1 0
Case #97: 0 0
Case #98: 1 1
Case #99: 0 0
Case #100: 1 1
